modules/simplytest_submission/src/SubmissionAccessControlHandler.php

```diff
@@ -43,7 +43,7 @@ protected function checkAccess(EntityInterface $entity, $operation, AccountInter
   protected function checkCreateAccess(AccountInterface $account, array $context, $entity_bundle = NULL) {
     $result = parent::checkCreateAccess($account, $context, $entity_bundle);
     if ($result->isNeutral()) {
-      return AccessResult::allowedIfHasPermission($account, 'add submission entities');
```

Comment:
Mainly because there is no 'add submission entities' permission :stuck_out_tongue_closed_eyes: we have not added any permissions yet.

We definitely should add some permissions for all the admin pages and operations, but I think we always want people to be able to create submissions. So either we just allow creation, or we create a permission and grant that to anonymous users during install.

I think permission review in general should be a follow up issue.